pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -337,6 +337,9 @@ func (o *InstallOptions) copyContent(ctx context.Context) error {
 	// Write secrets, config maps and pod to disk
 	// This does not need timeout, instead we should fail hard when we are not able to write.
 	podFileName := o.PodConfigMapNamePrefix + ".yaml"
+	if err := os.Remove(path.Join(resourceDir, podFileName)); err == nil {
```

Comment:
is `resourceDir` our target directory? If yes then it is revisioned and shouldn't contain any data, right?

Reply:
@p0lyn0mial i think this is for the pod config map... in theory, we don't need to remove the existing file, does not hurt probably?